about.html

```diff
@@ -0,0 +1,84 @@
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1" charset="utf-8">
+        <link rel="stylesheet" href="stetics.css"> 
+        <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300&family=Meera+Inimai&display=swap" rel="stylesheet">
+        <link rel="shortcut icon" href="img/github2.png" type="image/x-icon">
+    </head>
+    <body>
+        <header class="dflex spacebetween">
+            <img class="menu" src="img/Threebars.png" alt="menu">
+            <div class="menudesktop">
+                <ul class="dflex">
+                    <li><a href="about.html" rel="noopener">About</a></li>
```

Comment:
- [ ] [OPTIONAL] I think the `about` here should rather be `Home` and a link linking back to the `Home` page, instead of about again. That is i think better. 🙂